### DISCUSSION TITLE
Always Ignore/Import Chapters

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#1889] [Support]
How do I choose to always ignore or import chapters?

**Comment by mifi** [comment]
you can ignore it with the advanced setting (checkbox) "Ask about importing chapters from opened file"

**Reply by Jay123210599** [reply]
What if I always want to import chapters?

**Reply by mifi** [reply]
that's not currently possible. you'd have to press the "enter" key each time in order to accept the import

**Reply by Jay123210599** [reply]
Can you make it possible?

**Reply by mifi** [reply]
i will convert this into a feature request
